.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+      - main  # Jab bhi tum main branch pe code push karoge, yeh trigger hoga
+
+jobs:
+  build:
+    runs-on: ubuntu-latest  # Yeh runner hoga jo tumhare code ko test karega
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2  # Ye step tumhara code GitHub Actions pe pull karega
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)